documentation/release-notes/topics/version_3.rst

```diff
@@ -3,6 +3,107 @@
 #########
 Version 3
 #########
+**********************************
+Hyperview 3.11 (February 15, 2023)
+**********************************
+This section covers significant changes and bug fixes in Hyperview 3.11.x since version 3.10.x.
+
+.. note:: This release has changes and improvements to the Data Collector software. It is strongly recommended to update your installed Data Collectors to the latest version to maintain an optimal monitoring and discovery experience.
+
+=============================================
+New Licensed Feature: Rack PDU Outlet Control
+=============================================
+Outlet Control is a new licensed feature that allows users to:
+
+- Power on
+- Power off
+- Power cycle
+
+Supported Rack PDU hardware.
```

Comment:
I think this should be "supported Rack PDU hardware.", if it is meant to be a continuation of the above sentence. Otherwise, it looks like a standalone sentence.